script/appJQ.js

```diff
@@ -50,10 +50,20 @@ $(function(){
     }
   });
 
+  //$('#slides > img:gt(0)').hide();
```

Comment:
I know it is nice to have the code there for later use in case you want to use it again, but the point of version control is that you do have access to the older code in an older version. It takes some getting use to but it is all there!